Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -472,8 +544,36 @@ extension _FileManagerImpl {
         }
         return extendedAttrs
     }
-    
+#endif
+
     func attributesOfItem(atPath path: String) throws -> [FileAttributeKey : Any] {
+#if os(Windows)
+        return try path.withNTPathRepresentation { pwszPath in
+            var faAttributes: WIN32_FILE_ATTRIBUTE_DATA = .init()
+            guard GetFileAttributesExW(pwszPath, GetFileExInfoStandard, &faAttributes) else {
+                throw CocoaError.errorWithFilePath(path, win32: GetLastError(), reading: true)
+            }
+
+            let size: UInt64 = (UInt64(faAttributes.nFileSizeHigh) << 32) | UInt64(faAttributes.nFileSizeLow)
+            let creation: Date = Date(timeIntervalSince1970: faAttributes.ftCreationTime.interval)
+            let modification: Date = Date(timeIntervalSince1970: faAttributes.ftLastWriteTime.interval)
+            return [
+                .size: size,
```

Comment:
Maybe we could still call `_writeFileAttributePrimitive(_:as:)` here just so we're consistent even if it doesn't really do anything (since we do that on Linux too)?

Reply:
I suppose that should be harmless, but I find that it does hurt readability somewhat.